### PR TITLE
Add description for issue 28239 fix

### PR DESCRIPTION
**Pull request by mozok**
In General fixes add description for https://github.com/magento/magento2/issues/28239
PR https://github.com/magento/magento2/pull/33452

## Purpose of this pull request

Update M2.4.3 release notes.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/release-notes/open-source-2-4-3.html

## Links to Magento source code

-  linked PR https://github.com/magento/magento2/pull/33452

### CODE REVIEW

File: src/guides/v2.4/release-notes/open-source-2-4-3.md [hunk 1]
```diff
@@ -704,6 +704,10 @@ We have fixed hundreds of issues in the Magento 2.4.3 core code.
 
 *  The image resizing process no longer halts for images in unsupported format. Previously, when `catalog:images:resize` encountered an unsupported image format, the process stopped and Magento displayed this error: `bin/magento catalog:images:resize Unsupported image format`.
 
+<!--- ENGCOM-9148-->
+
+*  The image resizing process now process images marked as hidden in admin. Also added `--skip_hidden_images` option to skip hidden images during `catalog:images:resize` command processing. [GitHub-28239](https://github.com/magento/magento2/issues/28239)
```
Comment (BarnyShergold) [inline src/guides/v2.4/release-notes/open-source-2-4-3.md:709]:
```suggestion
*  The image resizing process now processes images marked as hidden in admin. Also added `--skip_hidden_images` option to skip hidden images during `catalog:images:resize` command processing. [GitHub-28239](https://github.com/magento/magento2/issues/28239)
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Small grammar correction.

File: src/guides/v2.4/release-notes/open-source-2-4-3.md (hunk 1 above)
Comment (mozok) [reply src/guides/v2.4/release-notes/open-source-2-4-3.md]: Thanks for review

**Review by BarnyShergold — APPROVED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Review by Den4ik — APPROVED** [review]

**Comment by mozok** [comment]
@BarnyShergold @dobooth Hi.
This PR has mistake. Referred fix (https://github.com/magento/magento2/pull/33452) was merged before release, but was not included in 2.4.3 release tag. It turns out I've mistakenly added information about fix on release page.
Also this comment https://github.com/magento/magento2/issues/28239#issuecomment-889023072 not quite true.

Should I create new PR to remove wrong info, or create revert of current PR?

**Comment by Den4ik** [comment]
@gabrieldagama Could you explain why https://github.com/magento/magento2/pull/33452 not released with 2.4.3?

**Comment by mozok** [comment]
Also I was noted, that there is no changes in https://github.com/magento/magento2/blob/2.4.3/app/code/Magento/MediaStorage/Console/Command/ImagesResizeCommand.php 
associated with https://github.com/magento/magento2/pull/33452/files

**Comment by gabrieldagama** [comment]
> @gabrieldagama Could you explain why [magento/magento2#33452](https://github.com/magento/magento2/pull/33452) not released with 2.4.3?

This was merged 27 days ago. Internally we have had branch cut off before that, this is necessary for the regression process. It will be released on 2.4.4.

**Comment by Den4ik** [comment]
Thanks for information. @mozok i think we should revert this commit

**Comment by mozok** [comment]
@Den4ik ok
I don't see usual 'Revert' option near commit message https://github.com/magento/devdocs/pull/9084#event-5200843704
Should I create new PR and delete text, that was added in this PR?

**Comment by Den4ik** [comment]
@mozok Yep, I mean create PR with revert commit

**Pull request merged**